src/components/common/EntityInfo.vue

```diff
@@ -60,7 +62,7 @@ export default class EntityInfo extends Mixins(EnumMixin) {
       case RouteNames.CORRECTION:
         return 'Correction - Incorporation Application'
       case RouteNames.ALTERATION:
-        return 'Alteration'
+        return 'Company Information'
```

Comment:
Should this getter be in common mixin also?

Reply:
Yea, i think so, will update!
I forgot to mention the changes to entity info in the PR. Thanks for the reminder!

Reply:
Updated in next commit